### PR TITLE
rework OpenCritic right column style

### PR DESCRIPTION
**Pull request by candela97**
Use the same style as existing metacritic sections so the fonts are no longer blurry and overall looks nicer, also removed some related CSS styles that I think are unused/redundant.
Screenshot comparison: https://imgur.com/a/PSxq5X7

### CODE REVIEW

File: css/enhancedsteam.css [hunk 1]
```diff
@@ -2299,23 +2299,6 @@ input[type=checkbox].es_dlc_selection:checked + label {
...
-/***************************************
- * Apps Store pages
- * add_metacritic_userscore()
- **************************************/
-#game_area_userscore {
-  color: #4a617c
-}
-#game_area_userscore span {
-  color: #7e96a3;
-}
```
[the remark this answers is not in the document]
Comment (candela97) [reply css/enhancedsteam.css]: I think these are leftovers from https://github.com/jshackles/Enhanced_Steam/commit/535209a9528251fc49d9f2e93383901cb428b209
which were later replaced by https://github.com/jshackles/Enhanced_Steam/commit/58d56ec0b53b8b9227e66a4946a2225b90a50622

**Comment by candela97** [comment]
I've added a fix for #468: now if there's no the metacritic section, the OpenCritic section should always appear in the same position.

File: css/enhancedsteam.css [hunk 2]
```diff
@@ -2299,23 +2299,6 @@ input[type=checkbox].es_dlc_selection:checked + label {
...
-#game_area_metascore.es_area_userscore .metacritic {
-  margin-top: -4px !important;
-}
-#game_area_metascore .score,
-#game_area_userscore .score {
-  padding: 0 !important;
-}
```
[the remark this answers is not in the document]
Comment (MxtOUT) [reply css/enhancedsteam.css]: image

You are still missing applying this to the metascore

File: css/enhancedsteam.css [hunk 3]
```diff
@@ -2347,6 +2330,7 @@ input[type=checkbox].es_dlc_selection:checked + label {
   font-size: 25px;
   text-align: center;
   float: left;
+  padding: 0 !important;
```
Comment (MxtOUT) [inline css/enhancedsteam.css]:
```suggestion
  padding: 0;
```
This selector is very specific, you don't need `!important`

File: css/enhancedsteam.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (candela97) [reply css/enhancedsteam.css]: derp, added and commented

**Pull request merged**